packages/manager/modules/pci/src/projects/project/storages/databases/database/service-integration/service-integration.html

```diff
@@ -5,33 +5,61 @@
     ></cui-message-container>
 
     <h2 data-translate="pci_databases_service_integration_tab_title"></h2>
-
-    <oui-message
-        data-ng-if="$ctrl.addableServicesList.length === 0"
-        data-type="warning"
-        data-dismissable
-    >
-        <span
-            data-translate="pci_databases_service_integration_tab_no_service"
-            data-translate-values="{engineName: $ctrl.engineName}"
-        ></span>
-    </oui-message>
-    <p
-        data-translate="pci_databases_service_integration_tab_descritpion"
-        data-translate-values="{engineName: $ctrl.engineName}"
-    ></p>
+    <p data-translate="pci_databases_service_integration_tab_descritpion"></p>
     <oui-datagrid
         class="mt-5"
         data-page-size="10"
         data-rows="$ctrl.serviceIntegrationList"
     >
         <oui-datagrid-column
-            data-title=":: 'pci_databases_service_integration_tab_service' | translate: {engineName: $ctrl.engineName}"
+            data-title=":: 'pci_databases_service_integration_tab_source_service' | translate"
             data-property="sourceServiceName"
             data-sortable="asc"
             data-type="string"
             data-searchable
             data-filterable
+        >
+            <span>{{$row.sourceServiceName}}</span>
+            <button
+                class="oui-link"
+                type="button"
+                ng-click="$ctrl.goToDatabase({id: $row.sourceServiceId})"
```

Comment:
```suggestion
                data-ng-click="$ctrl.goToDatabase({id: $row.sourceServiceId})"
```

There are others "data-" missing, I'll let you add them :)

Reply:
Added missing `data-`: https://github.com/ovh/manager/pull/7186/commits/083c8a9628024ea1f5ec362602d344d16ee0de4c